test/setuptest.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+#####################################################################
+# Obtain the various ReferenceDesignator values from asset-management
+#####################################################################
+
+# set up links to the CSVs in asset-management/deployment
+ls ../../asset-management/deployment/*.csv | awk -F"/" '{print "ln -s ../../asset-management/deployment/" $NF " " $NF}' > amdeploy
+source amdeploy
+
+# extract the ReferenceDesignator column values from all these CSVs
```

Comment:
move this to the python code (parsing of the files). It's appropriate to have environment settings specified in the shell script, but it shouldn't be processing data.

Reply:
The only things remaining in the shell script now are the steps necessary to establish soft links to the asset-management/deployment CSVs and preload_database.sql